src/api/app/helpers/webui/notification_helper.rb

```diff
@@ -0,0 +1,11 @@
+module Webui::NotificationHelper
+  def show_all_link
```

Comment:
What about `link_to_all`? I think its easier to understand

Reply:
Yes, changed!